books/free-programming-books-id.md

```diff
@@ -126,6 +127,10 @@
 * [Tutorial Pemrograman PHP untuk Pemula](https://www.petanikode.com/tutorial/php) - Ahmad Muhardian (Petani Kode) *(Dalam Proses)*
 
 
+### Lumen
+
+* [Lumen Fundamental](https://santrikoding.com) - Yudi Purwanto(HTML) [(PDF)](https://santrikoding.com/account/ebook/lumen-fundamental) _(Membutuhkan akun santrikoding atau email yang valid)_
```

Comment:
One link is enough, use the most specific and select right format: HTML or PDF

See also CONTRIBUTE format guidelines to solve lint errors